packages/mjml-text/README.md

```diff
@@ -19,32 +30,36 @@ This tag allows you to display text in your email.
  ```
 
 <p align="center">
-  <a href="https://mjml.io/try-it-live/components/text">
-    <img width="100px" src="https://mjml.io/assets/img/svg/TRYITLIVE.svg" alt="try it live" />
+  <a href="https://mjml.io/try-it-live/elements/text">
+    <img width="100px" src="https://mjml.io/assets/img/svg/TRYITLIVE.svg"
+         alt="try the code live" />
   </a>
-</p>
+</p> 
 
 <aside class="notice">
-  `MjText` can contain any HTML tag with any attributes. Don't forget to encode special characters to avoid unexpected behaviour from MJML's parser
+Don't forget to encode special characters to avoid unexpected
+  behaviour from MJML's parser.
 </aside>
 
- attribute                    | unit          | description                                 | default value
```

Comment:
I have a small concern about this too, if we change the way we describe unit we should modify this for all the docs and harmonize everything.

Reply:
As same for HTML, we expect MJML developer to be aware of CSS attribute. So I don't think we should be as verbose here

Reply:
Shortened.

Reply:
Let's revert this one for this PR, we can still discuss on slack about this one for a future improve but as stated the linter will provide accurate enough results for now

Reply:
I'm not confident what part of the file this conversation refers to.

Maybe you want to revert all changes to the attributes table. I'll do it.

Maybe you want me to streamline the table. I tried. Lines 34-53.

Reply:
As said if we want to tweak those descriptions like that, then we need to do the same for everywhere.